.github/workflows/chaos-validate.yml

```
@@ -0,0 +1,58 @@
+      - 'api/**'
+      - 'config/**'
+      - 'go.mod'
+      - 'go.sum'
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+jobs:
+  chaos-validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)